examples/cromfs/cromfs/testdir2/TheThreeLittlePigs.txt

```diff
@@ -14,4 +14,4 @@ The little pig saw the wolf climb up on the roof and lit a roaring fire in the f
 
```

Comment:
This is just a test file, but in this case, the trailing newline is for visual spacing since the story of the 3 little pigs is displayed.  It really needs the trailing newline since all of the text is double spaced and without the final newline, the text will run together on the display.